README.md

```diff
@@ -1 +1,25 @@
-# HelloWorld
\ No newline at end of file
+# Hello World
```

Comment:
- [ ] [OPTIONAL] Your pull request should have a summary describing introduced changes in more detail, please include the changes in the PR description

Reply:
Hi @Ismailco,
The description is the first comment of the PR?. I'll summarize every step in the process.

Reply:
@raminka13, Yes, the first comment of the PR. you can add more information on what you did in this PR